lib/include/prjxray/xilinx/xc7series/frames.h

```diff
@@ -0,0 +1,38 @@
+#ifndef PRJXRAY_LIB_XILINX_XC7SERIES_FRAMES_H
+#define PRJXRAY_LIB_XILINX_XC7SERIES_FRAMES_H
+
+#include <string>
+#include <vector>
+
+#include <absl/strings/str_split.h>
+#include <prjxray/xilinx/xc7series/configuration.h>
+#include <prjxray/xilinx/xc7series/frame_address.h>
+#include <prjxray/xilinx/xc7series/part.h>
+
+namespace prjxray {
+namespace xilinx {
+namespace xc7series {
+
+// Contains frame information which is used for the generation
+// of the configuration package that is used in bitstream generation.
+class Frames {
+       public:
+	typedef std::vector<uint32_t> FrameData;
+	typedef std::map<FrameAddress, FrameData> Frames2Data;
+
+	int readFrames(const std::string& frm_file_str);
```

Comment:
Please add method comments.  What does "addMissingFrames" do, and why would I call it?  Etc.

Reply:
I actually added the comments to the cc files